RedHatInsights/patchman-ui
# Tweak template assignment wizard table

# Description

Mockup: https://www.sketch.com/s/598ac9a1-16ce-47b1-8515-e79928a2b2fb/a/eKRw04p

Tweak template assignment wizard table:
- add sorting to Systems table inside Template creation modal
- display 'N/A' instead of blank cell when OS version in not present for a system
- TableView: Skeleton table now has the same number of columns as the table it's representing

# How to test the PR

Verify the first above 2 bullet points in template assignment wizard.
Verify the last bullet point on all Patchman tables.


# Checklist:
